editor/editor_export.cpp

```diff
@@ -1163,10 +1163,13 @@ void EditorExport::add_export_preset(const Ref<EditorExportPreset> &p_preset, in
 String EditorExportPlatform::test_etc2() const {
 
 	String driver = ProjectSettings::get_singleton()->get("rendering/quality/driver/driver_name");
-	bool etc2_supported = ProjectSettings::get_singleton()->get("rendering/vram_compression/import_etc");
+	bool etc_supported = ProjectSettings::get_singleton()->get("rendering/vram_compression/import_etc");
+	bool etc2_supported = ProjectSettings::get_singleton()->get("rendering/vram_compression/import_etc2");
 
-	if (driver == "GLES2" && !etc2_supported) {
-		return TTR("Target platform requires 'ETC' texture compression for GLES2. Enable support in Project Settings.");
+	if (driver == "GLES2" && !etc_supported) {
+		return TTR("Target platform requires 'ETC' texture compression for GLES2. Enable 'rendering/vram_compression/import_etc' in Project Settings.");
+	} else if (driver == "GLES3" && !etc2_supported && !etc_supported) {
+		return TTR("Target platform requires 'ETC2' texture compression for GLES3. Enable 'rendering/vram_compression/import_etc2' in Project Settings.");
```

Comment:
I think if you export with GLES3, most platforms only ask for etc2_support but not etc1, so the third check is unnecesary

Reply:
Fixed. I removed the `etc` feature from GLES3 on Android then.

Reply:
what if driver fallback is set to best?
then in some case, etc1 is needed.
in that case, we just enable etc1 too on setting?

Reply:
Yeah I just thought about that. This might have broken the fallback system for games indeed, I'll see how to improve it.

Reply:
I opened #26569 so that we don't forget about it.